tf/tfprocess.py

```diff
@@ -186,6 +186,14 @@ def restore(self, file):
         print("Restoring from {0}".format(file))
         self.saver.restore(self.session, file)
 
+    def process_loop(self, batch_size, test_batches):
+        # Get the initial steps value in case this is a resume from a step count
+        # which is not a multiple of total_steps.
+        steps = tf.train.global_step(self.session, self.global_step)
+        total_steps = self.cfg['training']['total_steps']
+        for _ in range(steps % total_steps, total_steps):
```

Comment:
Maybe this is clearer as `range(total_steps - (steps % total_steps))`? (Same result either way)